File: R.Entities/Entities/ViewModels/UserModel.cs
using R.Entities.Entities;

namespace R.Entities.Entities.ViewModels
{
    public class UserModel : AppUser
    {
        public UserModel()
        {
        }

        public UserModel(AppUser input)
        {
            if (input != null)
            {
                Id = input.Id;
                FirstName = input.FirstName;
                LastName = input.LastName;
                AvatarUrl = input.AvatarUrl;
                UniquePath = input.UniquePath;
            }
        }
    }
}

File: R.WebUI/ClientApp/src/shared/models/user-model.ts
export class UserModel {
	id: string;
	userName: string;
	password: string;
	firstName: string;
	lastName: string;
	accessToken: string;
	avatarUrl: string;
	tokenExpirationInTimeStamp: number;

	uniqueName?: string;
	confirmedPassword?: string;
	dateOfBirth?: Date;

	constructor(init?: Partial<UserModel>) {
		if (init) {
			Object.assign(this, init);
		}
	}
}